From 56e52128bf5be1b55233825bc8ee36e8e818587b Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Wed, 18 Dec 2024 12:49:04 -0500
Subject: [PATCH 1/1] gps: izat: Disable slim_daemon

---
 izat.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/izat.conf b/izat.conf
index 2585a8a..e361ad5 100644
--- a/izat.conf
+++ b/izat.conf
@@ -213,7 +213,7 @@ VENDOR_ENHANCED_PROCESS=1
 
 PROCESS_NAME=slim_daemon
 PROCESS_ARGUMENT=
-PROCESS_STATE=ENABLED
+PROCESS_STATE=DISABLED
 PROCESS_GROUPS=gps oem_2901 can plugdev diag sensors
 PREMIUM_FEATURE=1
 IZAT_FEATURE_MASK=0xf0
-- 
2.43.0

